src/components/common/buttons/ClaimButton.js

```diff
@@ -1,14 +1,36 @@
 // @flow
-import React from 'react'
-import { Platform } from 'react-native'
+import React, { useCallback } from 'react'
+import { Animated, Platform, View } from 'react-native'
 import { PushButton } from '../../appNavigation/PushButton'
 import { withStyles } from '../../../lib/styles'
 
-const ClaimButton = ({ screenProps, styles }) => (
-  <PushButton routeName="Claim" testID="claim_button" screenProps={screenProps} style={styles.claimButton}>
-    Claim
-  </PushButton>
-)
+const ClaimButton = ({ screenProps, styles, animated }) => {
+  const [containerSize, setContainerSize] = React.useState([])
+
+  const Button = (
+    <PushButton
+      routeName="Claim"
+      testID="claim_button"
+      screenProps={screenProps}
+      style={[styles.claimButton, { transform: containerSize }]}
+      contentStyle={{ marginHorizontal: -16 }}
```

Comment:
why is it -16? Also, please put this into a stylesheet

Reply:
Is about a default margin from the library, and that cut the text on Native. `MarginHorizontal : 0` not works.

Reply:
Then you should use -theme.sizes.defaultDouble